wordle-app/pages/index.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <v-container>
+    <v-card class="text-center">
+      <v-alert v-if="game.gameState != GameState.Playing" :color="game.gameState == GameState.Won ? 'success' : 'error'"
+        class="mb-5" tile>
+        <h3>
+          You've
+          {{ game.gameState == GameState.Won ? "Won! 🥳" : "Lost... 😭" }}
+        </h3>
+        <v-card-text>
+          The word was: <strong>{{ game.secretWord }}</strong>
+        </v-card-text>
+        <v-btn variant="outlined" @click="game.startNewGame()">
+          <v-icon size="large" class="mr-2"> mdi-restart </v-icon> Restart Game
+        </v-btn>
+      </v-alert>
+      <v-card-title v-else>Wordle</v-card-title>
+
+      <v-card-text class="d-flex  flex-row justify-end">
+        <v-chip color="secondary" class="mr-2" @click="showUserNameDialog=true">
+          <v-icon class="mr-2" > mdi-account </v-icon> {{ nameUserNameDialog }}
+        </v-chip>
+        <v-chip color="secondary" class="mr-2">
+          <v-icon class="mr-2"> mdi-timer </v-icon> {{ stopwatch.seconds }}
+        </v-chip>
+      </v-card-text>
```

Comment:
This looks sick and professional. Definitely thinking about implementing this.

Reply:
Thank you.